src/main/features/claimant-response/views/defendants-response.njk

```diff
@@ -148,11 +148,8 @@
           {% endif %}
         {% elseif page === 1 %}
           {% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.INSTALMENTS %}
-            {{
-              repaymentPlanBlock(
-                heading = 'How they want to pay',
-                repaymentPlan = claim.response.paymentIntention.repaymentPlan
-              )}}
+            <p>{{ t('The defendant suggested this repayment plan:') }}</p>
+            {{ repaymentPlanBlock(repaymentPlan = claim.response.paymentIntention.repaymentPlan) }}
```

Comment:
isn't it better to leave heading in macro which can be reused?

Reply:
this isn't a heading. this is just a paragraph. with the heading here it is duplicated both at the top of the page and at the. if you look at the screen shot you will see how it looks.